iphone/metadata/de-DE/subtitle.txt

```diff
@@ -1 +1 @@
-GPS Navigation, Wandern
```

Comment:
Important keywords were lost. They were not duplicated in keywords list. This change filtered out all searches of "GPS maps" or "GPS navigation" in the store.